server/src/main/java/org/apache/kafka/network/RequestConvertToJson.java

```diff
@@ -623,6 +629,8 @@ public static JsonNode response(AbstractResponse response, short version) {
                 return DeleteGroupsResponseDataJsonConverter.write(((DeleteGroupsResponse) response).data(), version);
             case DELETE_RECORDS:
                 return DeleteRecordsResponseDataJsonConverter.write(((DeleteRecordsResponse) response).data(), version);
+            case DELETE_SHARE_GROUP_OFFSETS:
```

Comment:
Please can you move `ALTER_SHARE_GROUP_OFFSETS` in this file (2 instances) to their correct places too? Thanks.